### PR TITLE
Add RemoteAttestation Demo

### PR DESCRIPTION
**Pull request by yilingxu**
Add a RemoteAttestation demo.

### CODE REVIEW

File: demos/RemoteAttestation/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+This project demonstrates how to run FAKE RemoteAttestation on hyperEnclave, which equipped with FAKE TPM and no CA support. The RemoteAttestation equipped with hardware TPM and CA are not open source now.
```
Comment (Bonjourz) [inline demos/RemoteAttestation/README.md]: How about:

which is equipped with FAKE TPM and is without the CA support?

**Comment by yilingxu** [comment]
你好，邮件已经收到，我会尽快给你回复。

**Comment by Bonjourz** [comment]
We may need to show that we have placed some demos in the "demos" directory in the "README.md". So how about adding the following description in README.md like:

We take `hello_c` as an example. (Command should be done inside Docker container):
```bash
$ cd /root/occlum/demos/hello_c

# Compile the user program with the Occlum toolchain
$ occlum-gcc -o hello_world hello_world.c
# Ensure the program works well outside enclave
$ ./hello_world
Hello World

# Initialize a directory as the Occlum instance, and prepare the Occlum's environment
$ mkdir occlum_instance && cd occlum_instance
$ occlum init
$ cp ../hello_world image/bin/
$ occlum build

# Run the user program inside an HyperEnclave's enclave via occlum run
$ occlum run /bin/hello_world
Hello World!

+ #### Other demos
+ More demos can be found in [demos](https://github.com/HyperEnclave/hyperenclave/tree/master/demos).
```

**Comment by yilingxu** [comment]
I have add a README.md at demos/RemoteAttestation/README.md. @Bonjourz

**Comment by yilingxu** [comment]
> > We may need to show that we have placed some demos in the "demos" directory in the "README.md". So how about adding the following description in README.md like:
> > We take `hello_c` as an example. (Command should be done inside Docker container):
> > ```shell
> > $ cd /root/occlum/demos/hello_c
> > 
> > # Compile the user program with the Occlum toolchain
> > $ occlum-gcc -o hello_world hello_world.c
> > # Ensure the program works well outside enclave
> > $ ./hello_world
> > Hello World
> > 
> > # Initialize a directory as the Occlum instance, and prepare the Occlum's environment
> > $ mkdir occlum_instance && cd occlum_instance
> > $ occlum init
> > $ cp ../hello_world image/bin/
> > $ occlum build
> > 
> > # Run the user program inside an HyperEnclave's enclave via occlum run
> > $ occlum run /bin/hello_world
> > Hello World!
> > 
> > + #### Other demos
> > + More demos can be found in [demos](https://github.com/HyperEnclave/hyperenclave/tree/master/demos).
> > ```
> 
> I have add a README.md at demos/RemoteAttestation/README.md.



> We may need to show that we have placed some demos in the "demos" directory in the "README.md". So how about adding the following description in README.md like:
> 
> We take `hello_c` as an example. (Command should be done inside Docker container):
> 
> ```shell
> $ cd /root/occlum/demos/hello_c
> 
> # Compile the user program with the Occlum toolchain
> $ occlum-gcc -o hello_world hello_world.c
> # Ensure the program works well outside enclave
> $ ./hello_world
> Hello World
> 
> # Initialize a directory as the Occlum instance, and prepare the Occlum's environment
> $ mkdir occlum_instance && cd occlum_instance
> $ occlum init
> $ cp ../hello_world image/bin/
> $ occlum build
> 
> # Run the user program inside an HyperEnclave's enclave via occlum run
> $ occlum run /bin/hello_world
> Hello World!
> 
> + #### Other demos
> + More demos can be found in [demos](https://github.com/HyperEnclave/hyperenclave/tree/master/demos).
> ```

File: demos/RemoteAttestation/README.md (hunk 1 above)
Comment (yilingxu) [reply demos/RemoteAttestation/README.md]: Thanks, done.

**Comment by Bonjourz** [comment]
I have tested on my platform, and get the result:


Seems that it works well.


@yilingxu Thanks for your contribution!

**Pull request merged**